internal/postprocessor/config.go

```diff
@@ -90,17 +91,24 @@ func (p *postProcessor) loadConfig() error {
 		c.GoogleapisToImportPath[v.InputDirectory] = &libraryInfo{
 			ServiceConfig: v.ServiceConfig,
 			ImportPath:    v.ImportPath,
+			RelPath:       v.RelPath,
 		}
 	}
 	for _, v := range owlBotConfig.DeepCopyRegex {
-		c.ClientRelPaths = append(c.ClientRelPaths, v.Dest)
 		i := strings.Index(v.Source, "/cloud.google.com/go")
 		li, ok := c.GoogleapisToImportPath[v.Source[1:i]]
 		if !ok {
 			return fmt.Errorf("unable to find value for %q, it may be missing a service config entry", v.Source[1:i])
 		}
-		li.ImportPath = v.Source[i+1:]
-		li.RelPath = v.Dest
+		if li.ImportPath == "" {
+			li.ImportPath = v.Source[i+1:]
+			fmt.Printf("set li.ImportPath from v.Source: %s", v.Source)
```

Comment:
This may be noisey, do we want these? If so, they should use log instead of fmt

Reply:
These were development debug statements that I forgot to remove. Fixed, thanks!